src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallerFactory.cs

```diff
@@ -63,6 +63,7 @@ internal class WorkloadInstallerFactory
         /// <returns>The <see cref="InstallType"/> associated with the SDK.</returns>
         public static InstallType GetWorkloadInstallType(SdkFeatureBand sdkFeatureBand, string dotnetDir)
         {
+            // TODO: does this file come with the SDK under dotnetDir? Or can it be under the userProfileDir for local installs?
```

Comment:
If this stuff gets installed to the user local folder, we'd need to add a `WorkloadInstall.IsUserLocal` here and use the user local folder instead.

Reply:
The .NET 6 installers from https://dotnet.microsoft.com/download/dotnet/6.0 don't seem to have any such files. Do you expect them to be there?

Reply:
Only the msi-based installers will currently have a file here, as if there's no marker file, it will be a file-based installation.  Also, we only added the MSI marker recently, so I'm not sure if it made it into Preview 7.  The daily RC1 builds from https://github.com/dotnet/installer should have it.